webapp/channels/src/components/with_tooltip/index.tsx

```diff
@@ -111,6 +114,11 @@ export default function WithTooltip({
     }
 
     const placements = useMemo<{initial: Placement; fallback: Placement[]}>(() => {
+        // if an explicit placement is provided, use it exclusively
+        if (placement) {
+            return {initial: placement, fallback: [placement]};
+        }
+
```

Comment:
can you explain the resoning behind adding an explicit placement

Reply:
with smaller text, the tooltip was preferably being displayed above the banner, even after specifying the direction as down. I had to add this to force it display in the desired position if needed.